services/horizon/internal/ingest/processors/operations_processor.go

```diff
@@ -351,8 +351,13 @@ func (operation *transactionOperationWrapper) Details() (map[string]interface{},
 		details["trustee"] = source.Address()
 		details["trustor"] = op.Trustor.Address()
 		details["authorize"] = xdr.TrustLineFlags(op.Authorize).IsAuthorized()
-		if xdr.TrustLineFlags(op.Authorize).IsAuthorizedToMaintainLiabilitiesFlag() {
-			details["authorize_to_maintain_liabilities"] = xdr.TrustLineFlags(op.Authorize).IsAuthorizedToMaintainLiabilitiesFlag()
+		authLiabilities := xdr.TrustLineFlags(op.Authorize).IsAuthorizedToMaintainLiabilitiesFlag()
+		if authLiabilities {
+			details["authorize_to_maintain_liabilities"] = authLiabilities
+		}
+		clawbackEnabled := xdr.TrustLineFlags(op.Authorize).IsClawbackEnabledFlag()
+		if clawbackEnabled {
+			details["claw_back_enabled"] = clawbackEnabled
```

Comment:
It's _clawback_ everywhere else.
```suggestion
			details["clawback_enabled"] = clawbackEnabled
```

Reply:
Good catch